server/chat-plugins/chat-monitor.js

```diff
@@ -110,6 +169,36 @@ Chat.registerMonitor('warn', {
 	},
 });
 
+Chat.registerMonitor('evasion', {
+	location: 'EVERYWHERE',
+	punishment: 'EVASION',
+	label: 'Filter Evasion Detection',
+	monitor(line, room, user, message, lcMessage, isStaff) {
+		let [word, reason] = line;
+		if (typeof word !== 'string') throw new Error(`evader monitor filters should only have strings`);
+		let regex = evasionFilterCache.get(word);
```

Comment:
this may be really premature, but why don't we just generate every regex on load and store it with the line instead of requiring a separate map lookup every time?

Reply:
Oh yeah we discussed this in DMs, iirc my answer was something along the lines of "i don't want to have to have cases for adding and changing filters, and complicate file loading"